src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const FEMALE = 'f';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+        {age
```

Comment:
Try using here '&&' instead of the ternary operators. Then there haven't to return 'null'

Reply:
> Try using here '&&' instead of the ternary operators. Then there haven't to return 'null'

Agree, it doesn't make sense to use ternary here